yarn.lock

```diff
@@ -473,7 +473,7 @@ callsites@^2.0.0:
   resolved "https://registry.yarnpkg.com/callsites/-/callsites-2.0.0.tgz#06eb84f00eea413da86affefacbffb36093b3c50"
   integrity sha1-BuuE8A7qQT2oav/vrL/7Ngk7PFA=
 
-chainpoint-proof-json-schema@^1.1.0:
+chainpoint-proof-json-schema@^1.2.0:
```

Comment:
Yarn.lock was never updated to 2.0.0... did you just manually update the package.json perhaps?

Reply:
The lockfile doesn't want to update until the dependency is published to NPM, otherwise it just installs an old version. I'll rerun the install command which will update the lockfile when I've published the updated schema package